aiida_vasp/io/wavecar.py

```diff
@@ -0,0 +1,37 @@
+"""Tools for parsing CHGCAR files."""
+
+from aiida_vasp.utils.aiida_utils import get_data_class
+from aiida_vasp.io.parser import BaseFileParser
+
+
+class WavecarParser(BaseFileParser):
```

Comment:
Unit tests to aiida_vasp/io/tests/test_wavecar_io.py, please

Reply:
Added unit tests and dummy WAVECAR file